whichone/static/js/game.js

```diff
@@ -1,6 +1,16 @@
 function stopGame() {
     stopped = true;
+
+    if (!muteSound) {
+        var audio = new Audio('/static/resources/gameover.mp3');
+        audio.volume = 0.3;
+        audio.play();
+    }
+
+    console.log("Game over. Final score: " + userCurrentScore);
```

Comment:
Remove dev logs from source control.

Same below

Reply:
This got removed and then somehow re-added... sorted.